lib/permalink.ts

```diff
@@ -50,7 +50,13 @@ class Permalink {
   }
 
   stringify(data) {
-    return this.rule.replace(rParam, (match, name) => data[name]);
+    return this.rule.replace(rParam, (match, name) => {
+      const descriptor = Object.getOwnPropertyDescriptor(data, name);
+      if (descriptor && typeof descriptor.get === 'function') {
+        throw new Error('Invalid permalink setting!');
```

Comment:
I think it can throw out the cause of the error, which makes it easier to locate the problem.

e.g.
```js
throw new Error(`xxxxx :${name}`);
```